File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/3 - 3 and 4 November 2018/0 - My Exam/03. Film Premiere .py
name = input()
packet = input()
count = int(input())

price = 0

if name == "John Wick":
    if packet == "Drink":
        price = count * 12
    elif packet == "Popcorn":
        price = count * 15
    elif packet == "Menu":
        price = count * 19
elif name == "Star Wars":
    if packet == "Drink":
        price = count * 18
    elif packet == "Popcorn":
        price = count * 25
    elif packet == "Menu":
        price = count * 30
elif name == "Jumanji":
    if packet == "Drink":
        price = count * 9
    elif packet == "Popcorn":
        price = count * 11
    elif packet == "Menu":
        price = count * 14

if name == "Star Wars" and count >= 4:
    price -= price * 0.3
if name == "Jumanji" and count == 2:
    price -= price * 0.15

print(f"Your bill is {price:.2f} leva.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/04. Online Exam - 15 and 16 June 2019/03. Film Premiere.js
function filmPremiere(params) {
    let movie = params.shift();
    let packet = params.shift();
    let countTickets = Number(params.shift());

    let prices = {
        'John Wick': {'Drink': 12, 'Popcorn': 15, 'Menu': 19},
        'Star Wars': {'Drink': 18, 'Popcorn': 25, 'Menu': 30},
        'Jumanji': {'Drink': 9, 'Popcorn': 11, 'Menu': 14}
    };

    let price = prices[movie][packet] * countTickets;

    if (movie == 'Star Wars' && countTickets >= 4) {
        price *= 0.7;
    } else if (movie == 'Jumanji' && countTickets == 2) {
        price *= 0.85;
    }

    console.log(`Your bill is ${price.toFixed(2)} leva.`);
}


filmPremiere(['John Wick', 'Drink', '6']);  // Expected Output:
// Your bill is 72.00 leva.

filmPremiere(['Star Wars', 'Popcorn', '4']);  // Expected Output:
// Your bill is 70.00 leva.

filmPremiere(['Jumanji', 'Menu', '2']);  // Expected Output:
// Your bill is 23.80 leva.
